src/pacmod3_node.cpp

```diff
@@ -65,6 +65,7 @@ ros::Publisher global_rpt_pub;
 ros::Publisher component_rpt_pub;
 ros::Publisher vin_rpt_pub;
 ros::Publisher turn_rpt_pub;
+ros::Publisher door_signal_rpt_pub;
```

Comment:
These should just be "door_rpt", not "door_signal_rpt." There is nothing being "signaled."